EIPS/eip-1363.md

```diff
@@ -181,24 +181,27 @@ interface ERC1363Spender {
    * approval. Return of other than the magic value MUST result in the
    * transaction being reverted.
    * Note: the token contract address is always the message sender.
-   * @param owner address The address which called `approveAndCall` function
-   * @param value uint256 The amount of tokens to be spent
+   * @param sender address The address which called `approveAndCall` function
+   * @param amount uint256 The amount of tokens to be spent
    * @param data bytes Additional data with no specified format
    * @return `bytes4(keccak256("onApprovalReceived(address,uint256,bytes)"))`
    *  unless throwing
    */
-  function onApprovalReceived(address owner, uint256 value, bytes memory data) external returns (bytes4);
+  function onApprovalReceived(address sender, uint256 amount, bytes memory data) external returns (bytes4);
 }
 ``` 
 
+Note that `transferAndCall`, `transferFromAndCall` MUST revert if the recipient is an EOA address, because EOA recipients do not implement the required ERC1363Receiver interface.  
+Note that `approveAndCall` MUST revert if the spender is an EOA address, because EOA spenders do not implement the required ERC1363Spender interface.
+
 ## Rationale
-The choice to use `transferAndCall`, `transferFromAndCall` and `approveAndCall` derives from the [ERC-20](./eip-20.md) naming. They want to highlight that they have the same behaviours of `transfer`, `transferFrom` and `approve` with the addition of a callback on receiver or spender.
+The choice to use `transferAndCall`, `transferFromAndCall` and `approveAndCall` derives from the [EIP-20](./eip-20.md) naming. They want to highlight that they have the same behaviours of `transfer`, `transferFrom` and `approve` with the addition of a callback on receiver or spender contracts.
```

Comment:
Small suggested tweak: "with the addition of a callback" => "with the addition of a required callback".

Also, I would add to the end of this paragraph an explanation for why this even matters:

> Requiring a callback on the receiver prevents tokens from accidentally being sent to contracts that do not know how to receive or spend them (which has caused the loss of millions of dollars worth of tokens previously). Requiring a callback on a spender contract ensures that intended spenders are notified that they have been granted an allowance to spend tokens on the sender's behalf, while ensuring that the spender supports this functionality.